app/controllers/support_interface/new_provider_users_controller.rb

```diff
@@ -0,0 +1,86 @@
+module SupportInterface
+  class NewProviderUsersController < SupportInterfaceController
```

Comment:
This a rubbish name but temporary. It is to distinguish this controller from the existing [ProviderUsersController](https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/controllers/support_interface/provider_users_controller.rb). Once we want to turn the feature permanently we can rename

Reply:
👍